runtimes/instance-rootfs/create-debian-12-disk.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -euf
+
+# Variables
+ROOTFS_FILE="./rootfs.btrfs"
+MOUNT_ORIGIN_DIR="/mnt/debian"
+MOUNT_DIR="/mnt/vm"
+IMAGE_URL="https://cloud.debian.org/images/cloud/bookworm/latest/debian-12-genericcloud-amd64.tar.xz"
+IMAGE_NAME="debian-12-genericcloud.tar.xz"
+IMAGE_RAW_NAME="disk.raw"
+
+# Cleanup previous run
+umount "$MOUNT_ORIGIN_DIR" || true
+umount "$MOUNT_DIR" || true
+rm -f "$ROOTFS_FILE"
+
+# Prepare directories
+mkdir -p "$MOUNT_ORIGIN_DIR"
+mkdir -p "$MOUNT_DIR"
+
+# Download Debian image
+echo "Downloading Debian 12 image"
+curl -L "$IMAGE_URL" -o "$IMAGE_NAME"
+
+# Create rootfs.btrfs file
+echo "Creating rootfs.btrfs file"
+dd if=/dev/zero of="$ROOTFS_FILE" bs=1G count=2
+mkfs.btrfs "$ROOTFS_FILE"
+mount "$ROOTFS_FILE" "$MOUNT_DIR"
+
+# Extract Debian image
+echo "Extracting Debian 12 image"
+tar xvf "$IMAGE_NAME"
+
+# Mount first partition of Debian Image
+LOOPDISK=$(losetup --find --show $IMAGE_RAW_NAME)
+partx -u $LOOPDISK
+mount "$LOOPDISK"p1 "$MOUNT_ORIGIN_DIR"
```

Comment:
Why not using `btrfs-convert $LOOPDISK"p1` instead of extracting the files and putting them in a new filesystem ?